README.md

```diff
@@ -72,3 +72,12 @@ or
 ```json
   npx lerna exec -- npm publish
 ```
+## Storybook
+1. Run all packages storybooks
+```json
+  yarn run storybook:all
+``` 
+1. Run single composed storybook which containing all storybooks
+```json
+  yarn run storybook-composition
+``` 
```

Comment:
It is best if we add a comment that stage 1 will open 2 storybooks tabs and the terminal will be captured to the servers.

and that we need to run stage 2 in a different terminal for a composed storybook while the other 2 are still running.

Reply:
done